plugins/secom_user_sim.py

```
@@ -108,6 +113,19 @@ def _sim_fluo_search(self, dlg):
                 logging.debug("Changed exposure time to %g s", exp)
 
                 # Wait a little while
+                if self._should_stop.wait(1):
+                    break
+
+                # Move the focus (back and forth, to make sure we never end too far)
+                focuser = fms.focuser
+                if focuser is not None:
+                    refocus_dist = random.uniform(-1e-6, 1e-6)  # m
```

Ruff (0.15.14)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)